pages/glossary/changes-in-content.md

```diff
@@ -14,13 +14,13 @@ A _change in the content_ of a [web page][] caused by an event occurs when, by c
 
 - **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
 - **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
-- **audible changes:** the audio reproduced by the [web page][] changes.
+- **audible changes:** the audio rendered by the [web page][] changes.
 
-If the [web page][] is reproducing time-based media, rendered pixels and reproduced audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered and reproduced if the event is not fired, with the ones that are rendered if the event is fired.
+**Assumptions:**
 
-For the purpose of this definition, it is assumed that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event.
-
-For the purpose of this definition, it is assumed that the changes happen instantly after the event firing and therefore the comparison between the page before and after the event firing can be made at any time after the firing of the event.
+- If the [web page][] is rendering time-based media, rendered pixels and audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered if the event is not fired, with the ones that are rendered if the event is fired.
```

Comment:
I don't think this is an assumption.
As in, it's not something that we suppose and that make things go wrong if it doesn't hold.
It's a precision on how to handle a specific situation.